views/tests.ejs

```diff
@@ -42,6 +42,11 @@
 <!-- TODO: https://github.com/foolip/mdn-bcd-collector/issues/407 -->
 <script src="/resources/json3.min.js"></script>
 <script src="/resources/harness.js"></script>
+
+<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.3.1/styles/stackoverflow-dark.min.css">
+<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.3.1/styles/stackoverflow-light.min.css" media="(prefers-color-scheme: light)">
+<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.3.1/highlight.min.js"></script>
```

Comment:
https://github.com/foolip/mdn-bcd-collector/issues/407 (see json3.min.js above) applies here too. Can you arrange for this script to be loaded only after tests have finished running?

Reply:
Sure thing!  Do you want me to make the changes for highlight.js here, or tackle both scripts at once?